pkg/gcp/actuator/actuator.go

```diff
@@ -861,6 +954,31 @@ func checkServicesEnabled(gcpClient ccgcp.Client, permList []string, logger log.
 	return serviceAPIsEnabled, nil
 }
 
+func validateWIFProviderSpec(providerSpec minterv1.GCPProviderSpec) error {
+	var errors []error
+	isEmptyProjectNumber := providerSpec.ProjectNumber == ""
+	isEmptyPoolID := providerSpec.PoolID == ""
+	isEmptyProviderID := providerSpec.ProviderID == ""
+	isEmptyServiceAccountEmail := providerSpec.ServiceAccountEmail == ""
```

Comment:
nit: I'm not a huge fan of splitting this out into variables... this can be done inline below while reducing lines of code without really affecting readability.
```
if providerSpec.ProjectNumber == "" {
```